remote/bitbucket/internal/client.go

```diff
@@ -152,6 +154,23 @@ func (c *Client) CreateStatus(owner, name, revision string, status *BuildStatus)
 	return c.do(uri, post, status, nil)
 }
 
+func (c *Client) GetPermission(full_name string) (*RepoPerm, error) {
+	out := new(RepoPermResp)
+	uri := fmt.Sprintf(pathPermissions, c.base, full_name)
+	err := c.do(uri, get, nil, out)
+
+	if err != nil {
+		return nil, err
+	}
+
+	if len(out.Values) == 0 {
+		err = errors.New(fmt.Sprint("no permissions in repository ", full_name))
```

Comment:
actually, you can just do `fmt.Errorf("no permissions in repository %s", fullName)`